packages/manager/modules/dbaas-logs/src/index.js

```diff
@@ -8,11 +8,13 @@ import '@ovh-ux/ng-ovh-doc-url';
 import { Environment } from '@ovh-ux/manager-config';
 
 import logs from './logs';
+import cuiDualList from './components/dual-list';
```

Comment:
I'm wondering if we can replace all those `cui` components by `ovh-ui-kit` ones as a next step